classes/fields/datetime.php

```diff
@@ -549,21 +562,19 @@ public function format_time( $options, $js = false ) {
 					$format = pods_v( static::$type . '_time_format_custom', $options, '' );
 				} else {
 					$format = pods_v( static::$type . '_time_format_custom_js', $options, '' );
+					// Already in JS format.
+					$js = false;
 
 					if ( empty( $format ) ) {
 						$format = pods_v( static::$type . '_time_format_custom', $options, '' );
-						$format = $this->convert_format( $format, array( 'source' => 'php' ) );
+						$js = true;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 6527003da3af0e611978a5f00f6d8afd</sub>